### DISCUSSION TITLE
프로퍼티 노출 관련 버그

### DISCUSSION THREAD

**Discussion by Cha-SeongHun** [naverz/zepeto-studio-kor#549] [Issue Report (Legacy)]
노출된 Transform 프로퍼티가 이미지에서 보시는 것처럼 표시가 되지 않습니다.
원래 노출이 잘 되어 동작하던 것이었고, 자고 일어나서 에디터를 여니 갑자기 발생하였습니다.

![image](https://user-images.githubusercontent.com/100827088/160555733-4cfb0906-81dc-4736-94f9-d6df14e8e271.png)

아래는 이 컴포넌트의 Debug화면입니다.
![image](https://user-images.githubusercontent.com/100827088/160556361-e81fccfc-dec9-4a2d-ab66-fd0bc3672f3b.png)

시스템 환경은 Windows11, Zepeto패키지 버전은 1.1.5입니다.

**Comment by x4cf** [comment]
배열을 퍼블릭으로 선언하고, 하나라도 값이 비어있으면 배열이 고장나는 버그가 있습니다 :(

스크립트를 삭제 후 컴퓨터를 재시동한다음 다시 작성하면 해결됩니다

**Reply by x4cf** [reply]
스크립트를 인스팩터에서 해제한다음 컴퓨터를 재시동하는 방법도 있어요 :D
